### PR TITLE
DHFPROD-5088:minor fixes to deploy users and display mapped data in e…

### PR DESCRIPTION
**Pull request by kkanthet**
…xplorer

#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x ] JIRA_ID included in all the commit messages
- [x ] PR title is in the format JIRA_ID:Title
- [ x] Rebase the branch with upstream
- [x ] Squashed all commits into a single commit
- [n/a ] Added Tests
  

- ##### Reviewer:

- [x] Reviewed Tests

### CODE REVIEW

File: examples/reference-entity-model/gradle-dhs.properties [hunk 1]
```diff
@@ -17,3 +17,6 @@ mlAppServicesPort=8010
 
 mlUsername=hub-developer
 mlPassword=password
+
+mlFlowOperatorRole=flowOperator
```
Comment (rjrudin) [inline examples/reference-entity-model/gradle-dhs.properties:21]: Yuck - were there things breaking when using DHF in DHS because these weren't overridden? I guess that's because we still use this for some default permissions.

File: examples/reference-entity-model/src/main/ml-config/security/users/hub-developer.json [hunk 2]
```diff
@@ -5,6 +5,7 @@
   "role": [
     "data-hub-developer",
     "hub-central-user",
+    "hub-central-entity-model-writer",
```
Comment (rjrudin) [inline examples/reference-entity-model/src/main/ml-config/security/users/hub-developer.json:8]: Note that this will be simplified soon once we have the hub-central-developer role. Will only need to add that then.

File: examples/reference-entity-model/steps/mapping/mapCustomersJSON.step.json [hunk 3]
```diff
@@ -10,7 +10,7 @@
   "collections": [
     "default-mapping"
   ],
-  "additionalCollections": [],
+  "additionalCollections": ["Customer","mapCustomersJSON"],
```
Comment (rjrudin) [inline examples/reference-entity-model/steps/mapping/mapCustomersJSON.step.json:13]: Can you change "collections" to be this array? Check to see if any other step depends on "default-mapping", but that's the right fix here.

File: examples/reference-entity-model/gradle-dhs.properties (hunk 1 above)
Comment (kkanthet) [reply examples/reference-entity-model/gradle-dhs.properties:21]: yes, when we deploy users its failing bcs of this. I think we said flowDeveloper and operator will continue to  work in 5.2.x

File: examples/reference-entity-model/src/main/ml-config/security/users/hub-developer.json (hunk 2 above)
Comment (kkanthet) [reply examples/reference-entity-model/src/main/ml-config/security/users/hub-developer.json:8]: Yeah, just added so PM can use hub-developer  has access to everything for now, when we get hub-central-developer-role, will remove this.

File: examples/reference-entity-model/steps/mapping/mapCustomersJSON.step.json (hunk 3 above)
Comment (kkanthet) [reply examples/reference-entity-model/steps/mapping/mapCustomersJSON.step.json]: Moved to collections.

File: examples/reference-entity-model/gradle-dhs.properties (hunk 1 above)
Comment (SameeraPriyathamTadikonda) [reply examples/reference-entity-model/gradle-dhs.properties:21]: You should also add mlFlowDeveloperRole=flowDeveloper

Comment (SameeraPriyathamTadikonda) [reply examples/reference-entity-model/gradle-dhs.properties:21]: My bad. I missed it

File: examples/reference-entity-model/steps/mapping/mapCustomersJSON.step.json [hunk 4]
```diff
@@ -8,7 +8,9 @@
   "selectedSource": "collection",
   "sourceQuery": "cts.collectionQuery(['loadCustomersJSON'])",
   "collections": [
-    "default-mapping"
+    "default-mapping",
```
Comment (rjrudin) [inline examples/reference-entity-model/steps/mapping/mapCustomersJSON.step.json:11]: Please remove "default-mapping" too, it's not needed for anything. We don't want to include anything that's not needed, as it'll raise the question of "Why is this here?"
